src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -108,6 +110,13 @@ where
                         }
                     }
                 },
+                _ = min_trigger_interval.tick() => {
+                    // Periodically trigger compaction for all compaction groups.
+                    for cg in self.hummock_manager.compaction_group_manager().compaction_groups().await {
```

Comment:
Could we trigger them in place instead of send message to channel?

Reply:
yes, it seems to make more sense to directly perform `pick_and_assign`

Reply:
Actually, no, I think its better to send message to channel. We don't want the periodically scheduled compaction groups to jump ahead of the queue in arbitrary order.

Reply:
> Could we trigger them in place instead of send message to channel?

What would be the benefit of in place trigger?

Reply:
Not for benefit. Just for readable..